config/flipt.schema.cue

```diff
@@ -129,6 +131,9 @@ import "strings"
 	}
 
 	#tracing: {
+		enabled?: bool | *false
+		exporter?: "jaeger" | *"jaeger"
```

Comment:
```suggestion
		enabled: bool | *false
		exporter: "jaeger" | *"jaeger"
```

I think you can forgo making this optional if there is a default.

Reply:
Actually with `exporter` you might be able to get away with just:
```
exporter?: jaeger
```
Which just says if you specify the key `exporter` it must be the value `"jaeger"`.
And drop the default.

Reply:
i'll prob just leave this as is for now since we will very shortly allow others besides `jaeger`